_episodes/04-recipe.md

```diff
@@ -194,13 +202,14 @@ Let's dissect what's happening here.
 > > ## Answers
 > >
 > > 1. The config file should be the one we edited in the previous episode,
-> >    something like `/home/<user>/.esmvaltool/config-user.yml`.
+> >    something like `/home/<username>/.esmvaltool/config-user.yml` or `./config-user.yml`.
```

Comment:
Or the one we created in the tutorial `~/esmvaltool_tutorial/config_user.yml`

Reply:
Made the change.